### PR TITLE
Add recipes for stack_data, executing, and pure_eval

### PR DESCRIPTION
**Pull request by alexmojaki**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

Merry Christmas! 🎄 

I'm submitting these packages so that they can eventually be dependencies of IPython once https://github.com/ipython/ipython/pull/11886 is merged.

### CODE REVIEW

File: recipes/executing/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: b180cd26e6bc23305a20a7cbd228a6bab94ae359e0a304dfc58425e3b0bdf6be
+
+build:
+  number: 0
```
Comment (npavlovikj) [inline recipes/executing/meta.yaml]: Can this recipe be `noarch`?

File: recipes/pure_eval/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 207facfb53f183e740be37d3e13f25968e16e7799d7725940b3ee008266903c1
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv "
+  skip: true  # [py<35]
```
Comment (npavlovikj) [inline recipes/pure_eval/meta.yaml]: I think this recipe can be `noarch`. In that case, please remove the `skip` line, and use `python >=3.5` in both `host` and `run` sections below.

File: recipes/stack_data/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: df93061c669cba1a2256689c7fb19603be99f96d46a107dc5306c11be847ea3c
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv "
+  skip: true  # [py<35]
```
Comment (npavlovikj) [inline recipes/stack_data/meta.yaml]: I think this recipe can be `noarch`. In that case, please remove the `skip` line, and use `python >=3.5` in both `host` and `run` sections below.

Comment (npavlovikj) [reply recipes/stack_data/meta.yaml]: Here is some information about `noarch` packages, https://conda-forge.readthedocs.io/en/latest/meta.html#building-noarch-packages.

**Comment by alexmojaki** [comment]
Thanks, didn't know about noarch.

**Comment by npavlovikj** [comment]
Thank you for the prompt fixes and contributing @alexmojaki 🎉

**Comment by alexmojaki** [comment]
And thank you for the prompt merge :)

I'm looking at the feedstocks now. [stack_data](https://github.com/conda-forge/stack_data-feedstock) has a couple of commits, but the build failed because it can't find it's dependencies pure_eval and executing. But the new feedstocks for those two have zero commits in them, so of course they aren't available. Is it just a matter of waiting some more? Or is the failed build somehow holding the others up?

**Comment by npavlovikj** [comment]
@alexmojaki , can you please submit another PR with the recipe for `executing` only? As you know, the feedstock for `executing` has not been created yet. I checked with the core team, and it looks like travis-ci has an issue and can only convert one feedstock in an hour. 
Since you have 3 recipes in this PR, I am worried that if I just re-run the build, we will have the same issue. Thus, opening another PR just with `executing` should be the best workaround.
Please use the same recipe as here, and feel free to ping me to look at it once it is done.

**Comment by npavlovikj** [comment]
@alexmojaki , sorry for the bother - I just saw that the feedstock for `executing` has been generated, so someone from the core team must have pushed the magic button :)
So, please ignore my previous comment.

**Comment by alexmojaki** [comment]
Great, thanks for everything! All 3 packages seem to be properly in conda-forge now.

**Pull request merged**